dm/pkg/terror/error_list.go

```diff
@@ -433,6 +433,7 @@ const (
 	codeSyncerParseDDL
 	codeSyncerUnsupportedStmt
 	codeSyncerGetEvent
+	codeSyncerGetCharsetMap
```

Comment:
I think we needn't add this extra error since in most case the cause fetch charset-map error  should be target database internal error or sql connect problem. Return general sql error should be enough.

Reply:
ok, i will remove this error.